.github/workflows/ci.yaml

```diff
@@ -27,13 +27,13 @@ jobs:
       - name: Install Vulkan SDK
         uses: humbletim/install-vulkan-sdk@v1.2
         with:
-          version: 1.4.309.0
+          version: 1.4.321.0
           cache: true
       - if: ${{ runner.os == 'Windows' }}
         name: Install Vulkan Runtime with SwiftShader (Windows)
         uses: NcStudios/VulkanCI@v1.2
         with:
-          sdkVersion: 1.4.309.0
+          sdkVersion: 1.4.321.0
```

Comment:
At the very least the Windows CI doesn't work, because this `NcStudios/VulkanCI` action is stuck on specific versions of the Vulkan SDK, not sure what to do about that - can we get SwiftShader from elsewhere?

Reply:
I only did shader toy to work around a windows bug in their fake GPU thing. Actions are updating (https://github.com/actions/runner-images/issues/12677) so maybe it isn't even needed now?

Reply:
Based on the CI failures for https://github.com/Rust-GPU/rust-gpu/pull/400, it looks like a HLSL syntax error in `wgpu`/Naga's output (specifically when nested arrays are involved), and having SwiftShader just means `wgpu`'s Vulkan backend is used instead.

Windows 2025 changes nothing, which doesn't surprise me, now that I've seen what's going on.
Will try installing the Vulkan SDK via [`jakoch/install-vulkan-sdk-action`](https://github.com/jakoch/install-vulkan-sdk-action) and see if that passes tests.